contributors/design-proposals/mount-options.md

```diff
@@ -0,0 +1,145 @@
+# Mount options for mountable volume types
+
+## Goal
+
+Enable Kubernetes users to specify mount options with mountable volumes
+such as  - `nfs`, `glusterfs` or `aws-ebs` etc.
+
+## Motivation
+
+We currently support network filesystems: NFS, Glusterfs, Ceph FS, SMB (Azure file), Quobytes, and local filesystems such as ext[3|4] and XFS.
+
+Mount time options that are operationally important and have no security implications should be suppported. Examples are NFS's TCP mode, versions, lock mode, caching mode; Glusterfs's caching mode; SMB's version, locking, id mapping; and more.
+
+## Design
+
+Mount options can be specified as `mountOption`  as part of PV. As an example:
+
+``` yaml
+ apiVersion: v1
+  kind: PersistentVolume
+  metadata:
+    name: pv0003
+  spec:
+    capacity:
+      storage: 5Gi
+    accessModes:
+      - ReadWriteOnce
+    persistentVolumeReclaimPolicy: Recycle
+    nfs:
+      path: /tmp
+      server: 172.17.0.2
+      mountOption: "hard,nolock,nfsvers=3"
```

Comment:
hmm, okay - It is purely aesthetic thing and I could be wrong, so if we use a list, mount options will look like:

```
mountOption: ["hard", "nolock", "nfsvers=3"]
```

Or in case when there is only one option, it will look like:

```
mountOption: ["volume-name=foobar"]
```

So even if we use a list here, we still have to do some string processing.  such as split by `=` for key/values.